app/assets/stylesheets/application.css

```diff
@@ -0,0 +1,209 @@
+/*
```

Comment:
- [ ] Kindly try to follow the provided design as much as possible:

![image](https://user-images.githubusercontent.com/57421823/201234326-5a2011b6-7ec6-44a3-9629-51191dea879f.png)

-[ ] The mobile version of the design is a little bit messy:

![image](https://user-images.githubusercontent.com/57421823/201234385-e86a4863-9cc4-4ac3-ae40-2d06adb94cb8.png)

![image](https://user-images.githubusercontent.com/57421823/201234520-bfd12483-75de-466c-9a2b-d10e0d0619eb.png)

Please, kindly improve it so it looks professional.  Remember, this is the final capstone project. 💪